src/components/forms/AcademicGrantsForm.tsx

```diff
@@ -0,0 +1,594 @@
+                    <Radio
+                      id='repeat-applicant-yes'
+                      size='lg'
+                      name='repeatApplicant'
+                      value='Yes'
+                      mr={8}
+                    >
+                      <PageText fontSize='input'>Yes</PageText>
+                    </Radio>
+
+                    <Radio
+                      id='repeat-applicant-no'
+                      size='lg'
+                      name='repeatApplicant'
+                      value='No'
+                      defaultChecked
+                    >
+                      <PageText fontSize='input'>No</PageText>
+                    </Radio>
+                  </Stack>
+                </RadioGroup>
+              </Field>
+            )}
+          />
+
+          <Controller
+            name='canTheEFReachOut'
+            control={control}
+            defaultValue={true}
+            render={({ field: { onChange, value } }) => (
+              <FormControl id='canTheEFReachOut-control'>
+                <FormLabel htmlFor='canTheEFReachOut'>
+                  <PageText display='inline' fontSize='input'>
+                    Is it OK for a member of the Ethereum Foundation to reach out to you (say, in
+                    regards to getting involved in other opportunities that may come up)?
+                  </PageText>
+                </FormLabel>
+
+                <RadioGroup
+                  id='canTheEFReachOut'
+                  onChange={value => onChange(value === 'Yes')}
+                  value={value ? 'Yes' : 'No'}
+                  fontSize='input'
+                  colorScheme='white'
+                  mt={4}
+                >
+                  <Stack direction='row'>
+                    <Radio
+                      id='canTheEFReachOut-yes'
+                      size='lg'
+                      name='canTheEFReachOut'
+                      value='Yes'
+                      defaultChecked
+                      mr={8}
+                    >
+                      <PageText fontSize='input'>Yes</PageText>
+                    </Radio>
+
+                    <Radio id='canTheEFReachOut-no' size='lg' name='canTheEFReachOut' value='No'>
+                      <PageText fontSize='input'>No</PageText>
+                    </Radio>
+                  </Stack>
+                </RadioGroup>
+              </FormControl>
+            )}
+          />
+
+          <TextAreaField
+            id='additionalInfo'
+            label="Do you have any questions about this grants round, or is there anything else
+            you'd like to share?"
+            helpText="Is there anything we didn't cover in the above questions? Feel free to add any
+            relevant links here. This is optional."
+          />
+
+          <Center mb={12}>
+            <Captcha />
+          </Center>
+
+          <Center>
+            <SubmitButton
+              isValid
```

Comment:
seems we are not checking if there are some errors or missing required inputs and this will allow to always submit, is that ok?

Reply:
So, this form is using Zod for all the form's validation and the submit button is always enabled. When the user clicks it, if there are any errors, you will be auto-scrolled to the field in question (check the `shouldFocusError` in `useForm`).